aggregation/swap_types_test.go

```diff
@@ -1,13 +1,12 @@
-package models
+package aggregation
 
 import (
+	"github.com/1inch/1inch-sdk-go/internal/helpers/consts/chains"
+	"github.com/1inch/1inch-sdk-go/internal/helpers/consts/tokens"
```

Comment:
Set your GoLand settings like this to do auto-grouping of the imports:

![image](https://github.com/1inch/1inch-sdk-go/assets/6529295/816a773d-709a-46d4-89ed-30b40391f5ce)

Reply:
👍